packages/pytest-simcore/src/pytest_simcore/docker_compose.py

```diff
@@ -94,13 +94,37 @@ def testing_environ_vars(env_devel_file: Path) -> EnvVarsDict:
     # ensure we do not use the bucket of simcore or so
     env_devel["S3_BUCKET_NAME"] = "pytestbucket"
 
+    # ensure OpenTelemetry is not enabled
+    env_devel |= {
+        x: "null"
+        for x in (
+            "AGENT_TRACING",

```

Comment:
Please note that these settings are only used inside `simcore_docker_compose` fixture. My initial approach was to set all of the settings ending in`_TRACING` to `null`.
The post against this approach was: `How do I figure out it's disabled?` -> pretty hard since you cannot search for `SERVICE_TRACING`  and figure out where its used.
Since the scope is pretty limited to only osparc services that are used for tests, I think this might be the better approach.
If something bothers you you specifically disable it.


The other approaches where every single service has testing disabled in contests was already applied to everywhere. So we should be OK like this.